config/tgl-cavs.toml

```diff
@@ -55,7 +55,7 @@ load_offset = "0x30000"
 count = 7
 	[[module.entry]]
 	name = "BRNGUP"
-	uuid = "61EB0CB9-34D8-4F59-A21D-04C54C21D3A4"
+	uuid = "2B79E4F3-4675-F649-89DF-3BC194A91AEB"
```

Comment:
Sorry, not following. Is this fixing a bug where UUIDs are not aligned between toml and the C code (discovered by Windows driver) ?

Reply:
This is PR is for tgl build on windows, so it is not related to v.19, right ?  

Let's talk about the layout of manifest.
extended manifest
extended module config (UUID comes from .toml file and it is for CAVS FW)
.....
.....
CSE header
module config  (UUID comes from SOF FW. It exists for a long time)

the two UUID don't match.  So Now change UUID in toml to the one in SOF FW, then UUID in both extended manifest and normal one are matched.